Build a WMAgent docker image

(issue description has been completely refactored) 
**Impact of the new feature**
WMAgent

**Is your feature request related to a problem? Please describe.**
This is one of the first steps towards running WMAgent in a containerized environment.

**Describe the solution you'd like**
With this issue, we are supposed to deliver:
* based on the outcome image from: https://github.com/dmwm/WMCore/issues/11565 , build a Dockerfile in CMSKubernetes to create an image for WMAgent core (no database dependencies). 
  * Some work has already been done on this: https://github.com/dmwm/CMSKubernetes/blob/master/docker/pypi/wmagent/Dockerfile
* WMAgent package needs to come from Pypi: https://pypi.org/project/wmagent/ 
* image creation needs to be part of our CD pipeline (already is)

**Describe alternatives you've considered**
None

**Additional context**
Depends on: https://github.com/dmwm/WMCore/issues/11565
The knowledge obtained must be propagated to: https://github.com/dmwm/WMCore/issues/11570
Part of the following meta issue: https://github.com/dmwm/WMCore/issues/11314

Even though we considered the GH actions workflow, there was still a detailed that was missed and the CMSKubernetes PR actually broke our workflow:
https://github.com/dmwm/WMCore/actions/runs/5427386861/jobs/9870613631#step:5:858

The important lines when building the docker image are:
```
  ---> 4389876fc8ff
Step 34/43 : ADD bin $WMA_DEPLOY_DIR/bin
ADD failed: file not found in build context or excluded by .dockerignore: stat bin: file does not exist
Error: Process completed with exit code 1.
```

I think the best option here would be to actually break the wmagent dockerfile into two (wmagent-base and wmagent), this way we can actually benefit from a more reproducible build as well. I might give it a try during the weekend.

--- a/.github/workflows/docker_images_template.yaml
+++ b/.github/workflows/docker_images_template.yaml
@@ -25,7 +25,8 @@ jobs:
 
       - name: Build image
         run: |
-          curl -ksLO https://raw.githubusercontent.com/dmwm/CMSKubernetes/master/docker/pypi/${{inputs.wmcore_component}}/Dockerfile
+          svn checkout https://github.com/dmwm/CMSKubernetes/trunk/docker/pypi/${{inputs.wmcore_component}}
+          cd ${{inputs.wmcore_component}}
           sed -i -e "s,ENV TAG=.*,ENV TAG=${{steps.get-ref.outputs.tag}},g" Dockerfile
           cat Dockerfile
           echo "Sleeping 5min to ensure that PyPi packages are available..."

